src/pages/patientView/clinicalTrialMatch/ClinicalTrialMatchTableOptions.tsx

```diff
@@ -325,60 +327,23 @@ class ClinicalTrialMatchTableOptions extends React.Component<
                             }}
                         >
                             <CreatableSelect
-                                components={components}
                                 isClearable
                                 isMulti={false}
-                                menuIsOpen={false}
-                                onInputChange={(
-                                    inputValue: any,
-                                    actionMeta: any
-                                ) => {
-                                    if (
-                                        inputValue !== null &&
-                                        inputValue !== ''
-                                    ) {
+                                components={components}

```

Comment:
Does this have any effect for you?
For me it does not disable the dropdown.

Reply:
Sorry, my mistake. I have confused it with the menuIsOpen option, but that one doesn't quite work with the current solution.